docs/data/date-pickers/date-picker/DatePickerValue.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import dayjs, { Dayjs } from 'dayjs';
+import Stack from '@mui/material/Stack';
+import { LocalizationProvider } from '@mui/x-date-pickers';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { Unstable_NextDatePicker as NextDatePicker } from '@mui/x-date-pickers/NextDatePicker';
+
+export default function DatePickerValue() {
+  const [value, setValue] = React.useState<Dayjs | null>(dayjs('2022-04-07'));
+
+  return (
+    <LocalizationProvider dateAdapter={AdapterDayjs}>
+      <Stack spacing={2} direction="row">
+        <NextDatePicker
+          label="Uncontrolled picker"
+          defaultValue={dayjs('2022-04-07')}
+        />
+        <NextDatePicker
+          label="Controlled picker"
+          value={value}
+          onChange={(newValue) => setValue(newValue)}
```

Comment:
To be honest, we very often recreate a method (same on the grid model setters) and I don't know why
Maybe we are just doing it because we always did it :laughing: 

No problem using the shorter version for me

Reply:
Maybe for pedagogical reasons. The long version explicitly says to newbies that the `onChange` is expecting a function which has the new value in argument. Moreover not sure every one is super familiar with setter accepting a function

Reply:
> Moreover not sure every one is super familiar with setter accepting a function

But we are not using this feature here right ?